### PR TITLE
Allow for the XMLRPC feature in rtorrent

### PR DESCRIPTION
**Pull request by awh-tokyo**
Adds the xmlrpc-c module and updates rtorrent module to use it if configured at install time.

### CODE REVIEW

**Comment by awh-tokyo** [comment]
Uh-oh.  It works on my system, but I'm completely new at this.  Any pointers on how to make it pass the tests?

**Comment by Florin65** [comment]
you can try to put aclocal or better  autoreconf -fi in the BUILD file.

**Comment by awh-tokyo** [comment]
Okay, well, I give up.  Take the pull request or don't; I'll just leave it in my zlocal.

File: distributed/rtorrent/DEPENDS [hunk 1]
```diff
@@ -1,3 +1,7 @@
 depends  libtorrent
 depends  ncurses
 depends  curl
+depends  cppunit
```
Comment (Ratler) [inline distributed/rtorrent/DEPENDS]: I'm pretty sure this PR will build just fine if you move "depends cppunit" to libtorrent/DEPENDS instead :)

**Comment by awh-tokyo** [comment]
Thanks.  Just tried that; let's see how it goes.  Just figured out how to see the output from the auto tester.

**Comment by lunar-linux-b01** [comment]
SUCCESS!

**Comment by stumbles** [comment]
Remove line 1 and 7 in the BUILD

**Comment by dagbrown** [comment]
I tried to build it with xmlrpc, and the rtorrent build failed thus:

`checking if curses tinfo library is linked properly... yes
checking pkg-config is at least version 0.9.0... yes
checking for libcurl... yes
checking for libtorrent... yes
checking for XMLRPC-C... failed
configure: error: Could not compile XMLRPC-C test.
Creating /var/log/lunar/compile/rtorrent-0.9.4.xz 
! Problem detected during BUILD
`

**Comment by awh-tokyo** [comment]
Okay, closing this for now and I'll reopen it once I get it building on a fresh install.

**Pull request closed without merging**